### ISSUE TITLE
Request - add film formats from National Archives to code table

### ISSUE THREAD

**Issue by wellerjes** [ArctosDB/arctos#8143]
Help us understand your request (check below):
- [ ] search or data request (help with SQL or provide the specific fields you are trying to retrieve for your collection)
- [ ] add a new Github user to the Arctos Users team
- [ ] a new barcode series
- [ ] a new term for data entry or management 
- [ ] a change or new report or label
- [ ] update to an Agent (split, merge, or other type of change)
- [ ] bulkload changes (to Agents, Identifiers, etc that are not already available as a bulkloading tool)
- [ ] new feature or otherwise a good idea to make Arctos even better
- [ ] quarantine taxon name and suggested replacement (please supply links)
- [x] other

**Describe what you're trying to do** 
The [attribute for format exists](https://github.com/ArctosDB/code-table-work/issues/13); now the [code table needs to be populated](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctformat).

Since the creation of the attribute, the link to the source has been updated. https://www.archives.gov/research/catalog/lcdrg/authority_lists/formatlist is no longer valid, the information now resides at https://www.archives.gov/research/catalog/lcdrg/authoritylists/format.

I am requesting that the formats on the National Archives list (copied below) be added to the code table.  Thank you!

Film: 1/4" MAG
scope note: 1/4-inch Sync Reel-to-Reel

Film: 1/4" MAGW
scope note: 1/4-inch Wild Sound Reel-to-Reel

Film: CRI
scope note: Color Reversal Internegative

Film: DNC
scope note: Duplicate Negative, Composite

Film: DNS
scope note: Duplicate Negative, Silent

Film: DNT
scope note: Duplicate Negative Track

Film: DRS
scope note: Duplicate Reversal, Silent

Film: FGMC
scope note: Fine Grain Master Composite

Film: FGMS
scope note: Fine Grain Master Silent

Film: INCK
scope note: Internegative, Composite, Color

Film: INSK
scope note: Internegative, Silent, Color

Film: INTPCK
scope note: Interpositive, Composite, Color

Film: INTPSK
scope note: Interpositive, Silent, Color

Film: MAG
scope note: Full Coat Magnetic Sound Track

Film: MAGS
scope note: Magnetic Tape Stripe Sound Track

Film: MAGSC
scope note: Magnetic Tape Stripe, Composite

Film: MPC
scope note: Master Positive, Composite

Film: MPCK
scope note: Master Positive, Composite, Color

Film: MPPC
scope note: Master Projection Print, Composite

Film: MPPCK
scope note: Master Projection Print, Composite, Color

Film: MPPS
scope note: Master Projection Print, Silent

Film: MPPSK
scope note: Master Projection Print, Silent, Color

Film: MPS
scope note: Master Positive, Silent

Film: MPSK
scope note: Master Positive, Silent, Color

Film: MPT
scope note: 35/16mm Master Positive Track

Film: MRCK
scope note: Master Reversal, Composite Color

Film: MRSK
scope note: Master Reversal, Silent, Color

Film: RCK
scope note: Reversal, Composite, Color

Film: RSK
scope note: Reversal, Silent, Color

Film: ONC
scope note: Original Negative, Composite

Film: ONCK
scope note: Original Negative, Composite, Color

Film: ONS
scope note: Original Negative, Silent

Film: ONSK
scope note: Original Negative, Silent, Color

Film: ONT
scope note: Original Negative Track

Film: ORC
scope note: Original Reversal, Composite

Film: ORCK
scope note: Original Reversal, Composite, Color

Film: ORS
scope note: Original Reversal, Silent

Film: ORSK
scope note: Original Reversal, Silent, Color

Film: PPC
scope note: Projection Print, Composite

Film: PPCK
scope note: Projection Print, Composite, Color

Film: PPS
scope note: Projection Print, Silent

Film: PPSK
scope note: Projection Print, Silent, Color

**Comment by wellerjes** [comment]
Additionally, please at a value for "unknown". Thanks!

**Comment by dustymc** [comment]
These need individual issues. (Even if the source wasn't clearly unstable!) I can do that (but it will take a while and I probably won't be very good at it, should anyone want to help...). Is there any priority order or do you have all of these waiting for CT values?

**Comment by wellerjes** [comment]
@dustymc We looked over our data again; the first link (https://www.archives.gov/research/catalog/lcdrg/authority_lists/formatlist) no longer exists and the new one that I copied does not actually meet our needs. We don't actually have the data or the ability to parse data from https://www.archives.gov/research/catalog/lcdrg/authoritylists/format to figure out how to match this up.

We've decided to move [film gauge](https://en.wikipedia.org/wiki/Film_gauge) measurements into dimensions for the time being, and will revisit this code table when we add more/different formats of audiovisual to Arctos. Thank you!
